cosmo_tester/framework/testenv.py

```diff
@@ -265,6 +265,11 @@ def setUpClass(cls):
     def tearDownClass(cls):
         pass
 
+    def assert_outputs(self, expected_outputs):
+        outputs = self.client.deployments.outputs.get(self.test_id)
```

Comment:
here again, dont assume self.test_id is the deployment id, allow for passing it and assign self.test_id as the default value of the deployment id